graphics.cfg

```diff
@@ -1,7 +1,21 @@
 % https://creativecommons.org/publicdomain/zero/1.0/
 
 \ProvidesFile{graphics.cfg}%
-  [2016/06/04 v1.11 sample graphics configuration]%
+  [2024/02/28 v1.12 sample graphics configuration]%
+
+% Using l3 would be nicer here, but this one also works with very old LaTeX
+\def\Gin@DeclareEngineClashs#1{%
```

Comment:
`\Gin@DeclareEngineClash`  (or `\Gin@DeclareEngineClashes` if plural but I think I'd use singular) (obviously that change everywhere only commenting here)